mmv1/third_party/terraform/resources/resource_compute_instance_group_manager.go.erb

```diff
@@ -815,10 +824,32 @@ func resourceComputeInstanceGroupManagerDelete(d *schema.ResourceData, meta inte
 		err = computeOperationWaitTime(config, op, project, "Deleting InstanceGroupManager", userAgent, d.Timeout(schema.TimeoutDelete))
 	}
 
+	// If unset in state set to default value
+	if d.Get("wait_for_instances_status").(string) == "" {
+		if err := d.Set("wait_for_instances_status", "STABLE"); err != nil {
```

Comment:
This block should be in `Read` and not `Delete` right?

Reply:
yeah this is my bad, I'll take things a little slower next time